_posts/blog/2021-04-08-whitakerlab-at-collabw21.md

```diff
@@ -0,0 +1,71 @@
+
+CollabW features a few keynotes but it is largely run as an [unconference event](https://unconference.net/unconferencing-how-to-prepare-to-attend-an-unconference/) with lightning talks, discussion sessions, mini workshops, and speed blogging events all relating to one or a few central themes.
+This feels radically different to traditional conference events as you are elevated from audience member to participant, giving you the opportunity to create something during the event and have discussions with a wide variety of other attendees.
+
+The final day features a [hack day](https://software.ac.uk/cw21/hack-day) where small teams can work on community-generated projects addressing a need they see in the research software community.
+Importantly, these do not have to be software projects to be eligible.
+Documentation and event planning projects, among others, have also featured at previous hack days.
+But most important of all, there are prizes up for grabs for the top 3 projects! :wink:
+
+## CollabW21
+
+The central themes of [CollabW21](https://software.ac.uk/cw21), hosted online from 30th March to 1st April 2021, were FAIR Research Software, Diversity and Inclusion, and Software Sustainability.
+
+Some of the highlights included an inspiring [keynote on Diversity and Inclusion](https://youtu.be/8viA4y1pz_8?t=1943) from [Dr Chonnettia Jones](https://www.msfhr.org/chonnettia-jones) and an unbelievably brave panel discussion on [Disability and Accessibility in Research Software](https://www.youtube.com/watch?v=65a8c06VHOY).
+
+## The Lab's Reflections
+
+At the lab meeting after the event, each member that attended was asked to produce a slide reflecting on their experience at CollabW21.
+They were asked to highlight something they really enjoyed, was unexpected, or completely different to past conference experiences (signified by the :rocket: emoji); and a topic that resonated with them, got them thinking, that they’d like to share/discuss with the lab (signified by the :thinking: emoji).
+You can see their slides below.
+
+![](../../images/WhitakerLab_CollabW_sgibson.jpg)
+
+![](../../images/WhitakerLab_CollabW_lcarter.jpg)
+
+![](../../images/WhitakerLab_CollabW_abennett.jpg)
+_Source link: https://www.krysburnette.com/blog/2019/1/22/belonging-a-conversation-about-equity-diversity-amp-inclusion_
+
+![](../../images/WhitakerLab_CollabW_ekaroune.jpg)
+
+![](../../images/WhitakerLab_CollabW_msharan.jpg)
+
+![](../../images/WhitakerLab_CollabW_pherterich.jpg)
+
+![](../../images/WhitakerLab_CollabW_amehonic.jpg)
+
+As you can see, themes around disability, diversity, inclusion and accessibility resonated particularly strongly within the group.
+But the unconference style felt very refreshing, if not a little exhausting!
```

Comment:
I wasn't sure you needed the "But...". I may have written "The unconference style felt very refreshing, though some members found it a little exhausting!"

Reply:
Could probably also say "We also found the unconference style..." ?

Reply:
Yes, I like that!

Reply:
```suggestion
We also found the unconference style felt very refreshing, if not a little exhausting!
```